src/main/resources/messages.yml

```diff
@@ -90,6 +91,7 @@ en:
   items_to_pickup: '&2{1} items left to pickup'
   items_to_smelt: '&2{1} items left to smelt'
   items_to_craft: '&2{1} items left to craft'
+  level_to_gain: '&2{1} level left to gain'
```

Comment:
Our convention is to use the plural form everywhere since this will be true most of the time.
Please change this for the other translations as well.
```suggestion
  level_to_gain: '&2{1} levels left to gain'
```